app/views/sessions/triage/show.html.erb

```diff
@@ -13,7 +13,7 @@
     <%= render AppSearchComponent.new(
           form: @form,
           url: session_triage_path(@session),
-          triage_statuses: %w[safe_to_vaccinate do_not_vaccinate delay_vaccination required],
+          triage_statuses: %w[required delay_vaccination do_not_vaccinate safe_to_vaccinate],
```

Comment:
I copied this from `app/views/programmes/patients.html.erb`. Was wondering if we could put it somewhere and share it between the two partials. Any ideas or just leave it for now?